### PR TITLE
contributing md added

### PR DESCRIPTION
**Pull request by SoerenPannier**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## Contributing to the code and documentation
+
+As you might have already noticed while browsing our repo we use branches to implement features or fix bugs. Our `master` branch corresponds to the current emdi version on cran. This allows us to develop larger features on the `dev` branch while
+maintaining the possibility of providing hotfixes to emdi on cran. Such hotfixes 
+on the `master` branch will ONLY be done by emdi's core development team. All other contributions should be done via branches from the `dev` branch. Any kind of enhancement, e.g. an improvement of the documentation or a new feature should be 
+done in branches starting with `feature\`  while `bug\` signals us a bugfix.
+
+If you contribute to code and documentation please also add a bullet point at the 
+`news.md` file, and tell people what has changed. Please, prepare your pull request 
+thoroughly. Work on a up-to-date version of `dev`. If there are merge conflicts
```
Comment (akreutzmann) [inline CONTRIBUTING.md:41]: work on an up-to-data

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+### On adding documentation
+
+THANKS! You have read all the above and still want to contribute GREAT! Please note, 
+we use `roxygen2` for our documentation, hence you will need to use the `#'` escape
+to add your documentation directly in the R file! All changes in the markdown files 
+in the `man` folder are overwritten when building the package.
+
+### On enhancing the code
+Still here? Amazing! 
+Just some notes regarding our style we use snake_case for practically everything,
```
Comment (akreutzmann) [inline CONTRIBUTING.md:62]: not yet the case, but should be our goal

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+If your enhancement of bug fix corresponds to a github issue, make sure to link 
+it in the merge request.
+
+### But I really want to add something big! 
+Contact us (Soeren.Pannier@fu-berlin.de or Ann-Kristin.Kreutzmann@fu-berlin.de) 
+and we will figure it out. It will probably include you forking our repo for development purposes. 
+
+
+## Anything else I need to know?
```
Comment (akreutzmann) [inline CONTRIBUTING.md:77]: Space after heading

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+#### Reporting a bug
+
+Did you notice a bug while using emdi? Please let us know so we can fix it and improve yours and others experience with emdi. To make sure we fully understand the issue and can reproduce the bug: Please include the following details in your report.
+
+-   Your operating system
+-   The R version you are using
+-   The emdi version you are using
+-   The steps necessary to reproduce the bug (if possible, in form of a minium working example)
+
+#### Suggesting an enhancement
```
Comment (akreutzmann) [inline CONTRIBUTING.md:22]: Space after heading

**Review by akreutzmann — COMMENTED** [review]
I have only minor changes but would happy about inclusion ;)

File: CONTRIBUTING.md (hunk 1 above)
Comment (SoerenPannier) [reply CONTRIBUTING.md:41]: done

File: CONTRIBUTING.md (hunk 3 above)
Comment (SoerenPannier) [reply CONTRIBUTING.md:77]: added

**Review by akreutzmann — APPROVED** [review]

**Pull request merged**